.github/ISSUE_TEMPLATE/proposal.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Event/Presentation/Happening Proposal
+about: Suggest what we could do at the dat event
+title: ''
+labels: 'proposal'
+assignees: ''
+
+---
+
+**What do you suggest we do?**
```

Comment:
Who is 'we'? Is the idea that people would want to present their own thing or run their own workshop? SO it would be 'what is the name of your session'?

Reply:
"We" was referring to the attendees, organizers and the person sending the commit. Trying to get into a community spirit.

Reply:
This isn't entirely clear